examples/langserve-sample/packages/rag_vector_store/rag_vector_store/chain.py

```diff
@@ -0,0 +1,95 @@
+import os
+from typing import List
+
+from langchain.schema import format_document
+from langchain_core.documents import Document
+from langchain_core.output_parsers import StrOutputParser
+from langchain_core.prompts import ChatPromptTemplate
+from langchain_core.prompts.prompt import PromptTemplate
+from langchain_core.pydantic_v1 import BaseModel
+from langchain_core.runnables import (
+    ConfigurableField,
+    RunnableParallel,
+)
+from langchain_core.runnables.utils import ConfigurableFieldSingleOption
+from langchain_openai import ChatOpenAI
+
+from zep_python import ZepClient
+from zep_python.langchain import ZepVectorStore
+
+ZEP_API_KEY = os.environ.get("ZEP_API_KEY", None)  # Required for Zep Cloud
+
+ZEP_COLLECTION_NAME = os.environ.get("ZEP_COLLECTION", "langchaintest")
+
+if ZEP_API_KEY is None:
+    raise ValueError(
+        "ZEP_API_KEY is required for Zep Cloud. "
+        "Remove this check if using Zep Open Source."
+    )
+
+zep = ZepClient(
```

Comment:
Please add api_url as an optional value pulled from the environment. You can even comment it out and add a comment that it is required for Zep Open Source. You can use this mechanism to target the dev cluster when you're testing.